### PR TITLE
Add openj9 layer config

### PR DESCRIPTION
**Pull request by kusumachalasani**
Signed-off-by: kusumachalasani <kchalasa@redhat.com>

### CODE REVIEW

File: manifests/autotune-configs/openj9-actuator-config.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+tunables:
+- name: MaxInlineLevel
+  description: 'maximum number of nested calls that are inlined. Default is 9'
+  value_type: integer
+  upper_bound: '50'
+  lower_bound: '9'
+  queries:
+  - datasource: 'prometheus'
+    query: 'jvm_memory_used_bytes{area="heap", $CONTAINER_LABEL$="", $POD_LABEL$="$POD$"}'
+  step: 1
```
Comment (dinogun) [inline manifests/autotune-configs/openj9-actuator-config.yaml:35]: Please move this to the GC policy as that makes more sense. Can you also make the change in hotspot as well

File: manifests/autotune-configs/openj9-actuator-config.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- name: gc
+  description: 'Garbage collection policy'
+  value_type: categorical
+  choices:
+    - gencon
+    - balanced
+    - optavgpause
+    - optthruput
+    - metronome
```
Comment (dinogun) [inline manifests/autotune-configs/openj9-actuator-config.yaml:48]: Does metronome work for all JDK's ? Or is it only for special builds?

File: manifests/autotune-configs/openj9-actuator-config.yaml (hunk 1 above)
Comment (kusumachalasani) [reply manifests/autotune-configs/openj9-actuator-config.yaml]: Done.

**Review by dinogun — COMMENTED** [review]
@kusumachalasani Have you tested both openj9 and hotspot images to make sure
1. The right layers are being matched with the right images
2. It is not detecting both layers at once

**Comment by kusumachalasani** [comment]
> @kusumachalasani Have you tested both openj9 and hotspot images to make sure
> 
> 1. The right layers are being matched with the right images
> 2. It is not detecting both layers at once

Yes, I did test by running TFB benchmark with hotspot and petclinic with openj9. Both of them detected the layers correctly.

File: manifests/autotune-configs/openj9-actuator-config.yaml (hunk 2 above)
Comment (kusumachalasani) [reply manifests/autotune-configs/openj9-actuator-config.yaml]: it is working for all builds.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**